pydocstyle.cfg

```diff
@@ -0,0 +1,15 @@
+[pydocstyle]
+# D100 Missing docstring in public module
+# D102	Missing docstring in public method
+# D103	Missing docstring in public function
+# D105	Missing docstring in magic method
+# D200	One-line docstring should fit on one line with quotes
+# D203	1 blank line required before class docstring
+# D205	1 blank line required between summary line and description
+# D212	Multi-line docstring summary should start at the first line
+# D213	Multi-line docstring summary should start at the second line
+# D400	First line should end with a period
+# D401	First line should be in imperative mood
+# D404	First word of the docstring should not be This
+ignore = D100,D102,D103,D105,D200,D203,D205,D212,D213,D400,D401,D404
```

Comment:
This overrides the default ignore list: if an error is ignored by default and this error is not listed here, it is now no longer ignored.
Is this intentionally?
Otherwise we should consider using `add-ignore`.